app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountController.scala

```diff
@@ -216,6 +216,19 @@ object BankAccountController {
       .transform[AccountNumber](s => AccountNumber(s.replaceAllLiterally(" ", "")), _.value)
       .verifying("invalid", e => accountNumberRegex.test(e.value))
 
+  final case class AccountName(value: String) extends AnyVal
+
+  object AccountName {
+    implicit val format: OFormat[AccountName] = Json.format[AccountName]
+  }
+
+  val accountNameRegex: Predicate[String] = """^[A-Za-z0-9\-',/& ]{1,150}$""".r.pattern.asPredicate()
```

Comment:
max size for the bank account name is 40 characters

Reply:
No fixed